tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -189,6 +189,57 @@ function ( $passed_order ) use ( $order ) {
+		$this->assertIsArray( $notification_as_arrays );
+
+		$this->mock_webhook_handler->expects( $this->once() )
+			->method( 'handle_deferred_payment_intent_succeeded' );
+
+		$this->mock_webhook_handler->process_deferred_webhook( 'payment_intent.succeeded', $data, $notification_as_arrays );
+
+		$this->assertIsObject( $captured_notification );
+		$this->assertSame( 'payment_intent.succeeded', $captured_notification->type );
+
+		remove_action( 'wc_stripe_webhook_received', $listener, 10 );
+	}
```

PHPMD (2.15.0)

[error] The method test_process_deferred_webhook_normalizes_array_notification_for_wc_stripe_webhook_received is not named in camelCase. (undefined)

(CamelCaseMethodName)